packages/features/settings/outOfOffice/CreateOrEditOutOfOfficeModal.tsx

```diff
@@ -20,30 +22,50 @@ import {
   UpgradeTeamsBadge,
 } from "@calcom/ui";
 
+import { OutOfOfficeTab } from "./OutOfOfficeEntriesList";
+
 export type BookingRedirectForm = {
   dateRange: { startDate: Date; endDate: Date };
   offset: number;
   toTeamUserId: number | null;
   reasonId: number;
   notes?: string;
   uuid?: string | null;
+  forUserId: number | null;
 };
 
 export const CreateOrEditOutOfOfficeEntryModal = ({
   openModal,
   closeModal,
   currentlyEditingOutOfOfficeEntry,
+  setOOOEntriesUpdated,
+  setOOOEntriesAdded,
 }: {
   openModal: boolean;
   closeModal: () => void;
   currentlyEditingOutOfOfficeEntry: BookingRedirectForm | null;
+  setOOOEntriesUpdated: Dispatch<SetStateAction<number>> | null;
+  setOOOEntriesAdded: Dispatch<SetStateAction<number>> | null;
 }) => {
   const { t } = useLocale();
-  const utils = trpc.useUtils();
+
+  const searchParams = useCompatSearchParams();
+  const oooType = searchParams?.get("type") ?? OutOfOfficeTab.MINE;
 
   const { data: listMembers } = trpc.viewer.teams.listMembers.useQuery({});
   const me = useMeQuery();
-  const memberListOptions: {
+  const forwardingToMemberListOptions: {
+    value: number;
+    label: string;
+  }[] =
+    listMembers
+      ?.filter((member) => (oooType === "mine" ? me?.data?.id !== member.id : oooType === "team"))
```

Comment:
I suggest using an enum instead of hardcoding

Reply:
updated